tests/unit/Mvc/Dispatcher/Helper/DispatcherTestInitializeReturnFalseController.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Phalcon\Test\Unit\Mvc\Dispatcher\Helper;
```

Comment:
I don't think it is actually a helper and it should be located here. How about `_tests/_data`?